diff --git a/node_modules/expo-camera/build/ExpoCamera.web.js b/node_modules/expo-camera/build/ExpoCamera.web.js
index 272e2e7..cee32be 100644
--- a/node_modules/expo-camera/build/ExpoCamera.web.js
+++ b/node_modules/expo-camera/build/ExpoCamera.web.js
@@ -10,7 +10,7 @@ import { useWebCameraStream } from './web/useWebCameraStream';
 import { useWebQRScanner } from './web/useWebQRScanner';
 const ExponentCamera = React.forwardRef(({ type, poster, ...props }, ref) => {
     const video = React.useRef(null);
-    const native = useWebCameraStream(video, type, props, {
+    const native = useWebCameraStream(video, props.facing, props, {
         onCameraReady() {
             if (props.onCameraReady) {
                 props.onCameraReady();
